packages/daimo-common/test/note.test.ts

```diff
@@ -0,0 +1,21 @@
+import assert from "node:assert";
+import test from "node:test";
+import { keccak256, verifyMessage } from "viem";
+
+import {
+  generateNoteSeedAddress,
+  getNoteClaimSignatureFromSeed,
+} from "../src/note.ts";
+
```

Comment:
recommend adding unit test coverage for new link format and backcompat from old to new

Reply:
done in daimoLink.test.ts